src/controllers/room.js

```diff
@@ -0,0 +1,106 @@
+
+const model = require('../database/models');
+
+// creating a room
+
+const createRoom = async (req, res) => {
+  try {
+    const room = await model.RoomModel.create(req.body);
+    if (room) {
+
+      return res.status(200).json({ room });
+    }
+
+  } catch (error) {
+    return res.status(500).json({error: error.message})
+  }
+};
```

Comment:
Please consider add some validation because I saw you can send a request with empty data and also you can send multiple request with the same data..... you can see here
<img width="1440" alt="Screen Shot 2021-01-18 at 15 16 55" src="https://user-images.githubusercontent.com/54619678/104923189-62153e00-59a4-11eb-92b3-4f382dcc3c3d.png">

Reply:
I think here any field can be empty and can be filled latter using edit enpoint